docs/source/conf.py

```diff
@@ -1,12 +1,17 @@
 # Configuration file for the Sphinx documentation builder.
 
 import os
+import pathlib
 import sys
 
+import yaml
+
 sys.path.insert(0, os.path.abspath('.'))
 sys.path.insert(0, os.path.abspath('../'))
 sys.path.insert(0, os.path.abspath('../../'))
 
+import prepare_github_markdown
```

Comment:
Should this be placed on L8

Reply:
The module will not be found without the path being added. : )